app/controllers/gov_one_controller.rb

```diff
@@ -1,3 +1,7 @@
 class GovOneController < ApplicationController
-  def info; end
+  layout 'hero'
+
+  def info
```

Comment:
Do you think it is possible to stick to RESTful CRUD actions in this controller at this point in time?